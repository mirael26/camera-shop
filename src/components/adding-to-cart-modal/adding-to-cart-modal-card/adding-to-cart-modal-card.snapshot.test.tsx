import { renderWithReduxAndRouter } from '../../../test/helpers/render-with-redux-and-router';
import { productMock } from '../../../test/mocks';
import AddingToCartModalCard from './adding-to-cart-modal-card';

test('AddingToCartModalCard component renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<AddingToCartModalCard onSuccess={jest.fn()}/>, { initialState: { cart: { addedToCartItem: productMock }}});
  expect(asFragment()).toMatchSnapshot();
});
