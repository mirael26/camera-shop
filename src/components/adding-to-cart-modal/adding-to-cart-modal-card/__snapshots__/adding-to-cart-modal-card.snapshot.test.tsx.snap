// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddingToCartModalCard component renders correctly 1`] = `
<DocumentFragment>
  <p
    class="title title--h4"
  >
    Добавить товар в корзину
  </p>
  <div
    class="basket-item basket-item--short"
  >
    <div
      class="basket-item__img"
    >
      <picture>
        <source
          srcset="img/content/orlenok.webp, img/content/orlenok@2x.webp 2x"
          type="image/webp"
        />
        <img
          alt="Фотоаппарат «Орлёнок»"
          height="120"
          src="img/content/orlenok.jpg"
          srcset="img/content/orlenok@2x.jpg 2x"
          width="140"
        />
      </picture>
    </div>
    <div
      class="basket-item__description"
    >
      <p
        class="basket-item__title"
      >
        Фотоаппарат Орлёнок
      </p>
      <ul
        class="basket-item__list"
      >
        <li
          class="basket-item__list-item"
        >
          <span
            class="basket-item__article"
          >
            Артикул:
          </span>
           
          <span
            class="basket-item__number"
          >
            O78DFGSD83
          </span>
        </li>
        <li
          class="basket-item__list-item"
        >
          Плёночная фотоаппарат
        </li>
        <li
          class="basket-item__list-item"
        >
          Любительский уровень
        </li>
      </ul>
      <p
        class="basket-item__price"
      >
        <span
          class="visually-hidden"
        >
          Цена:
        </span>
        19 970 ₽
      </p>
    </div>
  </div>
  <div
    class="modal__buttons"
  >
    <button
      class="btn btn--purple modal__btn modal__btn--fit-width"
      type="button"
    >
      <svg
        aria-hidden="true"
        height="16"
        width="24"
      >
        <use
          xlink:href="#icon-add-basket"
        />
      </svg>
      Добавить в корзину
    </button>
  </div>
  <button
    aria-label="Закрыть попап"
    class="cross-btn"
    type="button"
  >
    <svg
      aria-hidden="true"
      height="10"
      width="10"
    >
      <use
        xlink:href="#icon-close"
      />
    </svg>
  </button>
</DocumentFragment>
`;
